packages/ui/src/lib/Modal.svelte

```diff
@@ -53,110 +70,106 @@
 	};
 </script>
 
-<!-- svelte-ignore a11y_no_noninteractive_tabindex -->
-<!-- svelte-ignore a11y_no_noninteractive_element_interactions -->
```

Comment:
I think we might still need this. Haven't checked out the branch locally, but Svelte should have complained about having a key listener on a div. 

Weird that the check passed, though 🤔

Reply:
It depends on the element's role. There's no need to add an ignore comment for elements with the "presentation" role. In this case, it makes sense because the backdrop element is interactive, but it's neither a button nor an input.

Reply:
I see, makes sense 👍